schemas/telemetry/_common-schema.json

```diff
@@ -92,6 +92,14 @@
           "pattern": "^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$"
         }
       }
+    },
+    "experimental_features":  {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏Do @DataDog/rum-mobile have a similar concept?

Reply:
We don't but this is something we could have eventually